tldw_Server_API/tests/External_Sources/test_reference_manager_contract.py

```
@@ -0,0 +1,168 @@
+from tldw_Server_API.app.api.v1.schemas.connectors import ConnectorSourceCreateRequest, SyncOptions
+from tldw_Server_API.app.core.External_Sources import connectors_service as svc
+from tldw_Server_API.app.core.External_Sources.reference_manager_adapter import ReferenceManagerAdapter
+from tldw_Server_API.app.core.External_Sources.reference_manager_types import (
+    NormalizedReferenceCollection,
+    NormalizedReferenceItem,
+    ReferenceAttachmentCandidate,
+)
+
+
+class _ReferenceManagerStub:
+    async def list_collections(self, account, *, cursor=None, page_size=100):  # pragma: no cover - contract stub
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `list_collections`

(ANN202)